tc/sgx/trusted_worker_manager/enclave/work_order_processor.h

```diff
@@ -21,65 +21,66 @@
 
 #include "parson.h"
 #include "types.h"
-#include "work_order_data_handler.h"
 #include "enclave_types.h"
+#include "work_order_data_handler.h"
 
 namespace tcf {
-        class WorkOrderProcessor {
-        public:
-            WorkOrderProcessor(EnclaveType in_enclave_type=SINGLETON_ENCLAVE) {
-                enclave_type = in_enclave_type;
-                ext_work_order_data = "";
-            }
-            ~WorkOrderProcessor();
-            ByteArray CreateErrorResponse(int err_code, const char* err_message);
-            ByteArray Process(EnclaveData* enclaveData, std::string json_str);
-            std::string ext_work_order_data;
-            EnclaveType enclave_type;
+    class WorkOrderProcessor {
+    public:
+        WorkOrderProcessor() {
+            ext_work_order_data = "";
+        }
+        ~WorkOrderProcessor();
+        ByteArray CreateErrorResponse(int err_code, const char* err_message);
+        ByteArray Process(EnclaveData* enclaveData, std::string json_str);
+        std::string ext_work_order_data;
```

Comment:
It's better to add a comment about ext_work_order_data. like "In case of singletion it's empty and in case of KME and WPE this contains additional data as mentioned in files work_order_processor_kme.h and work_order_processor_wpe.h respectively"

Reply:
Done